style.css

```diff
@@ -280,14 +347,16 @@ body {
   border: 1px solid #6171ff;
   box-sizing: border-box;
   border-radius: 8px;
+  color: #6070ff;
+  transition: color 0.1s;
 }
 
 .p-button:hover {
```

Comment:
- [ ] I kindly suggest that you add the button interaction for when your button is active.
### EXAMPLE:

**for pressed/active state**
```
button:active {
   background-color: #2230D2;
}
```